pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1152,6 +1229,22 @@ class SWAP(Operation):
 
     batch_size = None
 
+    @property
+    def pauli_rep(self):
+        if self._pauli_rep is None:
+            self._pauli_rep = qml.pauli.PauliSentence(
+                {
+                    qml.pauli.PauliWord({self.wires[0]: "I", self.wires[1]: "I"}): 0.5,
```

Comment:
Could do
```suggestion
                    qml.pauli.PauliWord({}): 0.5,
```
here, as empty slots in a `PauliWord` are the same as putting an identity in. However, the current version is nice and symmetric, so I'll leave it to you/reviewer #2 to decide on this one :)